accounts/urls.py

```diff
@@ -7,8 +7,10 @@
 from .views import profile
 from .views import SignUpView
 from .views import unsubscribe
+from django.contrib.auth import views as auth_views
 
 urlpatterns = [
+    path('login/', auth_views.LoginView.as_view(redirect_field_name='next_page'), name='login'),
```

Comment:
Looks like you may not have `pre-commit` installed which is because it's not in the contributing docs :upside_down_face: [Install pre-commit on your machine](https://pre-commit.com/#install), then run `pre-commit install` in the project directory. It'll apply our style guidelines when you attempt to commit something.

For example, it would have automatically changed the `'` to `"`. That particular rule makes it easier to search for a string such as `"login"` because you don't have to search for both `"login"` and `'login'` in the codebase.

Reply:
I opened https://github.com/djangonaut-space/wagtail-indymeet/issues/310 to address the inadequate documentation

Reply:
I was a bit scared to use pre-commit because it's not mentiond in the docs. 
Is the use of `pre-commit run --all-files` not allowed, right?

Reply:
That's my fault when I added it I didn't mention it. Don't worry about running it for all files. By default it's limited to the ones you're changing and that's good enough.

If we missed a number of files, that's something to be cleaned up in a separate PR